src/Directory/ShippingMethodCodeDirectory.php

```diff
@@ -38,7 +38,7 @@ public function getValues(): array
         $values = [];
         /** @var ShippingMethodInterface $method */
         foreach ($this->shippingMethodRepository->findAll() as $method) {
-            $values[$method->getName()] = $method->getCode();
+            $values[$method->getCode()] = $method->getCode();
```

Comment:
I don't agree with this change because we want to display the name and not the code in the settings form